### PR TITLE
fix: multi modal eg using CRUD REST

### PR DESCRIPTION
**Pull request by rutujasurve94**
update Jina to 1.1.0
modify flow yamls to enable REST API
add index_rest that indexes docs using the /index REST API
add index_rest to click options
change README to point towards /search (NOT /api/search)

### CODE REVIEW

File: multimodal-search-tirg/app.py [hunk 1]
```diff
@@ -33,6 +36,36 @@ def config():
     os.environ['JINA_PORT'] = '45678'
 
 
+def index_restful(num_docs):
+    f = Flow().load_config('flows/index.yml')
+
+    with f:
+        data_path = os.path.join(os.path.dirname(__file__), os.environ.get('JINA_DATA_FILE', None))
+        print(f'Indexing {data_path}')
```
Comment (cristianmtr) [inline multimodal-search-tirg/app.py:44]: remove print statements

**Review by cristianmtr — CHANGES_REQUESTED** [review]
small thing

File: multimodal-search-tirg/app.py (hunk 1 above)
Comment (rutujasurve94) [reply multimodal-search-tirg/app.py]: done

File: multimodal-search-tirg/README.MD [hunk 2]
```diff
@@ -85,6 +85,21 @@ This will start the server, where you can then run your query and see the result
...
 
+### With REST API
+
+```sh
+python app.py -t query_restful
+```
+
+Then:
+
+```sh
```
Comment (cristianmtr) [inline multimodal-search-tirg/README.MD:96]: Is this relevant for the multimodal search? This curl is using text

**Comment by nan-wang** [comment]
This PR needs further improvement and we are working on 1.2 anyway.

**Pull request closed without merging**